main/main.js

```diff
@@ -0,0 +1,52 @@
+const math = require ("./math-module.js")
+const string = require ("./strings-module.js")
+
+const {
+    sum,
+    multiply,
+    divide,
+    square
+} = math
+
+const {
+    firstChar,
+    compare,
+    reverse,
+    middleChars
+} = string
+
```

Comment:
you can do this object destructoring and point directly to the require statement.